hnn_core/network.py

```diff
@@ -288,6 +288,10 @@ class Network(object):
     legacy_mode : bool
         Set to True by default to enable matching HNN GUI output when drives
         are added suitably. Will be deprecated in a future release.
+    mesh_shape : tuple of int
+        Defines the (n_x, n_y) shape of the grid of pyramidal cells.
+        Default: None (use values from params which is a 10x10 grid).
+
```

Comment:
Can we actually get rid of this dependency on `Network._params` altogether?

Reply:
as in hard code it as a default?

Reply:
Yes. Isn't our goal to move away from using that big params dictionary?